english/dream/swim/swim.md

```diff
@@ -10,4 +10,4 @@ When you recover from the shock of your emotional 180 you find yourself alone.
 A pain shoots up your legs. You glance down to where your feet should be only to see a single gummy 
 flipper wrapped tightly in the ends of licorice whips.
 
-You are trapped. Your dream is going horribly wrong. You have been ensnared by... 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.